src/webapp/src/client/app/common/collapsable-menu/collapsable-menu.component.js

```diff
@@ -0,0 +1,35 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('simoonaApp.Common')
+        .component('aceCollapsableMenu', {
+            replace: true,
+            transclude: true,
+            bindings: {
+                menuName: '<',
+                menuIconClass: '<',
+                menuTextTranslation: '<',
+                isCollapsed: '<'
+            },
+            templateUrl: 'app/common/collapsable-menu/collapsable-menu.html',
+            controller: collapsableMenuController,
+            controllerAs: 'vm'
+        });
+
+    function collapsableMenuController() {
+        const vm = this;
+        const localStorageName = vm.menuName + 'isCollapsed';
```

Comment:
For possible conflicting names issues it's good enough, but prefix (something like `Menu`) would add also add more clarity for next generations. :)

Reply:
Added prefix.